client/src/GlobalStyle.js

```diff
@@ -0,0 +1,22 @@
+import { createGlobalStyle } from "styled-components/macro";
+
+const GlobalStyle = createGlobalStyle`
+  *,
+  *::after,
+  *::before {
+    box-sizing: border-box;
+  }
+
+  html {
+    font-size: 16px;
+    background-color: #eef5f9;
```

Comment:
You might consider css variables for this so we get the color it represents by the name.

Reply:
Yes, I won't add them to this project. But for other projects, I totally agree with that. 
Another solution is to have a [theme provider](https://styled-components.com/docs/advanced#theming).